client/src/components/hero/Hero.tsx

```diff
@@ -22,7 +22,7 @@ const UnstyledHero: FunctionComponent<HomeProps> = ({ className }) => {
     <Box className={className}>
       <Box className="hero" display="flex" alignItems="center">
         <Container>
-          <Box display="flex" justifyContent="center" alignItems="center">
+          <Box display="flex" justifyContent="flex-start" alignItems="center">
```

Comment:
why was this changed?

Reply:
I thought it was old version of code that's why my remote dir has previous value
Now I am working on latest version of code from master